### ISSUE TITLE
Fix low contrast of `colorAccent` on the night themes

### ISSUE THREAD

**Issue by BrayanDSO** [ankidroid/Anki-Android#12020]
Accessibility scanner complains on both "Dark" and "Black" themes about low contrast.

Can be seen by using the Accessibility scanner on any of the preferences' screens.

My suggestion is using `#42A5F5`, aka Material Blue 400, since it has a contrast > 4.5 on both "Dark" (bg: `#303030` and "Black" (bg: `#000000`) themes

**Comment by mikehardy** [comment]
I trust your judgement on this, if you've already used the scanner and proposed a color, you're on the frontier of knowledge on this one
